src/rasmus_fuel/realistic.py

```diff
@@ -0,0 +1,289 @@
+import numpy as np
+import numpy.typing as npt
+
+VESSEL_PARAM = {
+    # """Define vessel parameters
+    #   Parameters
+    # --------------
+    #   vessel_supersurface_area: float
+    #       area of the above water vessel structure exposed to wind [m ** 2]. Defaults to 345
+    #   vessel_subsurface_area: float
+    #       an average area of the lateral projection of underwater vessel structure [m ** 2]. Defaults to 245
+    #   vessel_water_drag_coefficient: float
+    #       water drag coefficient for vessel [kg/m]. Defaults to 6000
+    #   vessel_waterline_width: float
+    #       width of vessel at the waterline in [m]. Defaults to 30
+    #   vessel_waterline_length: float
+    #       length of vessel at the waterline in [m]. Defaults to 210
+    #   vessel_total_propulsive_efficiency: float
+    #       total propulsive engine efficiency. Defaults to 0.7
+    #   vessel_draught: float
+    #       vessel draught in [m]. Defaults to 11.5
+    #   vessel_specific_fuel_consumption: float
+    #       specific fuel consumption  [g/kWh]. Defaults to 180
+    #   vessel_DWT: float
+    #       vessel dead weight in kg. Defaults to 33434
+    #   vessel_conversion_factor_fuelmass2CO2: float
+    #       conversion factor from fuel consumption to mass of CO2 emmitted (diesel/gas oil). Defaults to 3.2060
+    # """
+    "vessel_waterline_width": 30.0,
+    "vessel_waterline_length": 210.0,
+    "vessel_total_propulsive_efficiency": 0.7,
+    "vessel_draught": 11.5,
+    "vessel_supersurface_area": 345.0,
+    "vessel_subsurface_area": 245.0,
+    "vessel_water_drag_coefficient": 6000.0,
+    "vessel_specific_fuel_consumption": 180.0,
+    "vessel_DWT": 33434.0,
+    "vessel_conversion_factor_fuelmass2CO2": 3.2060,
+    "vessel_reference_froede_number": 0.12,
+}
+
+PHYSICS_PARAM = {
+    # """Define physical model parameters
+    #     Parameters
+    # --------------
+    #     air_mass_density: float
+    #        mass density of air [kg/m**3]. Defaults to 1.225
+    #     wind_resistance_coefficient: float
+    #        wind resistance coefficent, typically in rages [0.4-1]. Defaults to 0.4
+    #     reference_froede_number: float
+    #        Froede number, dimensionless. Defaults to 0.12
+    #     spectral_average: float
+    #        spectral and angular dependency factor, dimensionless. Defaults to 0.5
+    #     surface_water_density: float
+    #        density of surface water [kg/m**3]. Defaults to 1029
+    #     acceleration_gravity: float
+    #        the Earth gravity accleration [m/s**2]. Defaults to 9.80665
+    #     """
+    "air_mass_density": 1.225,
+    "wind_resistance_coefficient": 0.4,
+    "spectral_average": 0.5,
+    "surface_water_density": 1029.0,
+    "acceleration_gravity": 9.80665,
+}
+
+
+def power_maintain_sog(
+    u_ship_og: npt.ArrayLike = None,
+    v_ship_og: npt.ArrayLike = None,
+    course_ship_og: npt.ArrayLike = None,
```

Comment:
@eshchekinova I have another question about the course over ground. Shouldn't we derive this from `u_ship_og` and `v_ship_og` rather than use a separate (and hence possibly inconsistent) variable? Or are we talking about heading here, which in my understanding is the direction the centerline of the vessel is pointing to?